api/tests/python/end_points/payments/test_payments.py

```diff
@@ -217,52 +207,50 @@ def test_get_transactions(client, jwt, app):
 
     payload = json.loads(response.data)
 
-    assert isinstance(payload.get('items'), list) is True
+    assert isinstance(payload.get('id'), int) is True
+    # TODO: Check invoices / receipts...
+    # assert isinstance(payload.get('invoices'), list) is True
 
+    return payload
 
-def test_create_transaction(client, jwt, app):
-    payment_id = 'abcd153'
-    request_uri = API_BASE_URI + payment_id + '/transaction'
 
-    test_params = [{
-        'redirect_uri': 'http://localhost'
-    }]
+@pytest.mark.skip
+def execute_complete_payment(client, payment, action):
+    """
+    Complete a payment.
+    :param client:
+    :param payment
+    :param action
+    :return:
+    """
+    headers = get_test_headers()
+
+    # PATCH /api/v1/payments/<int:nr_id>/payment/<int:payment_id>/<string:payment_action>
+    request_uri = API_BASE_URI + str(payment.get('nrId')) + '/payment/' + str(payment.get('id')) + '/' + action
+    test_params = [{}]
 
     query = build_test_query(test_params)
     path = build_request_uri(request_uri, query)
     log_request_path(path)
 
-    response = client.post(path)
-    payload = json.loads(response.data)
-
-    assert isinstance(payload.get('items'), list) is True
-
+    response = client.patch(path, data={}, headers=headers)
 
-def test_update_transaction(client, jwt, app):
-    payment_id = 'abcd153'
-    receipt_number = 'abcd'
-    transaction_id = 'test'
-    request_uri = API_BASE_URI + payment_id + '/transactions'
-
-    test_params = [{}]
-
-    query = build_test_query(test_params)
-    path = build_request_uri(request_uri, query)
-    body = json.dumps({
-        "receipt_number": receipt_number,
-        "transaction_identifier": transaction_id
-    })
-    log_request_path(path)
+    assert response.status_code == 200
 
-    response = client.put(path, json=body)
     payload = json.loads(response.data)
 
-    assert isinstance(payload.get('items'), list) is True
+    return payload
 
 
-def test_get_receipt(client, jwt, app):
-    payment_id = 'abcd153'
-    request_uri = API_BASE_URI + payment_id + '/receipt'
+@pytest.mark.skip
```

Comment:
looks like you altered this quite a bit, but are skipping it? Did you mean to take this line out after finishing?

Reply:
I have some additional work to do on receipt. I will do it as part of UPDATE NAMES ON RECEIPT HEADER FOR DRAFT NRs #5318 ticket.